UI/LobbyMapUI.cs

```diff
@@ -68,16 +87,34 @@ public class LobbyMapUI : Entity {
         private Rectangle mapBounds;
 
         private bool focused;
+        private bool closing;
 
         private bool openedWithRevealMap;
+        private readonly bool viewOnly;
+        private Vector2 initialPlayerCenter;
+        private float missingKeybindsTimeRemaining;
 
         #endregion
 
-        public LobbyMapUI() {
+        public LobbyMapUI(bool viewOnly = false) {
             Tag = Tags.PauseUpdate | Tags.HUD;
             Depth = Depths.FGTerrain - 2;
             Visible = false;
 
+            if (lobbyMapJoystick == null) {
+                lobbyMapJoystick = new VirtualJoystick(
+                    CollabModule.Instance.Settings.PanLobbyMapUp.Binding,
+                    CollabModule.Instance.Settings.PanLobbyMapDown.Binding,
+                    CollabModule.Instance.Settings.PanLobbyMapLeft.Binding,
+                    CollabModule.Instance.Settings.PanLobbyMapRight.Binding, Input.Gamepad, 0.1f);
+                lobbyMapUpButton = new VirtualButton(CollabModule.Instance.Settings.PanLobbyMapUp.Binding, Input.Gamepad, 0f, 0.4f);
+                lobbyMapDownButton = new VirtualButton(CollabModule.Instance.Settings.PanLobbyMapDown.Binding, Input.Gamepad, 0f, 0.4f);
+                lobbyMapLeftButton = new VirtualButton(CollabModule.Instance.Settings.PanLobbyMapLeft.Binding, Input.Gamepad, 0f, 0.4f);
+                lobbyMapRightButton = new VirtualButton(CollabModule.Instance.Settings.PanLobbyMapRight.Binding, Input.Gamepad, 0f, 0.4f);
```

Comment:
since those are static and only instantiated the first time, does it still work if you change your bindings after seeing the map once? 🤔

Reply:
Hmm good point, I think the Binding property is modified rather than replaced.  Pretty sure I tested that but I can do a sanity check tonight.

Reply:
Just tested and yes it updates with the new bindings.  Honestly I might just remove the static caching anyway just in case.  I think they should be lightweight enough.